app/controllers/hubstats/deploys_controller.rb

```diff
@@ -47,15 +51,35 @@ def find_net_additions(deploy_id)
       return total_additions - total_deletions
     end
 
+    # finds the total number of additions for all pull requests in this deploy
```

Comment:
The logic of `find_additions` and `find_deletions` belongs in your Deploy model.  Then you can call them directly on your Deploy instance like `@deploy.total_additions` and `@deploy.total_deletions`.